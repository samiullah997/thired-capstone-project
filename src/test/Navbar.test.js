import Nav from '../components/Navbar';

it('test navbar component', async () => {
  expect(<Nav />).toMatchSnapshot();
});
