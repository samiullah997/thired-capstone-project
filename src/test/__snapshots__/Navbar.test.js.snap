// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test navbar component 1`] = `<Navbar />`;
